scrape/scrape.go

```diff
@@ -149,6 +220,7 @@ func init() {
 	prometheus.MustRegister(targetScrapeSampleOutOfOrder)
 	prometheus.MustRegister(targetScrapeSampleOutOfBounds)
 	prometheus.MustRegister(targetScrapeCacheFlushForced)
+	prometheus.MustRegister(targetMetadataCache)
```

Comment:
Maybe refactor this to call MustRegister once with all the collectors

Reply:
Indeed. This code has been written before the variadic `MustRegister` was created.

Reply:
done!